src/dos/cdrom_image.cpp

```diff
@@ -38,12 +43,32 @@
 #include <string.h>
 #endif
 
+#if defined(WORDS_BIGENDIAN)
+#define IS_BIGENDIAN true
+#else
+#define IS_BIGENDIAN false
+#endif
+
 using namespace std;
 
 #define MAX_LINE_LENGTH 512
 #define MAX_FILENAME_LENGTH 256
 
+#ifdef DEBUG
+char* get_time() {
+	static time_t rawtime;
+	struct tm* ptime;
+    static char time_str[] = "00:00:00";
+
+	time(&rawtime);
+    ptime = localtime(&rawtime);
+    sprintf(time_str, "%02d:%02d:%02d", ptime->tm_hour, ptime->tm_min, ptime->tm_sec);
```

Comment:
I would use `snprintf` here. Just to be prevent an issue if someone will change formatting string, but forget to adjust `time_str`.

Reply:
Great suggestion, will make this change.